dotnet/samples/Concepts/Agents/ChatCompletion_Whiteboard.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Azure.AI.OpenAI;
+using Azure.Identity;
+using Microsoft.Extensions.AI;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Memory;
+
+namespace Agents;
+
+#pragma warning disable SKEXP0130 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+
+/// <summary>
+/// Demonstrate creation of <see cref="ChatCompletionAgent"/> and
+/// adding whiteboarding capabilities, where the most relevant information from the conversation is captured on a whiteboard.
+/// This is useful for long running conversations where the conversation history may need to be truncated
+/// over time, but you do not want to agent to lose context.
+/// </summary>
+public class ChatCompletion_Whiteboard(ITestOutputHelper output) : BaseTest(output)
+{
+    private const string AgentName = "FriendlyAssistant";
+    private const string AgentInstructions = "You are a friendly assistant";
+
+    /// <summary>
+    /// Shows how to allow an agent to use a whiteboard for storing the most important information
+    /// from a long running, truncated conversation.
+    /// </summary>
+    [Fact]
+    private async Task UseWhiteboardAsync()
```

Comment:
UseWhiteboardToStoreInformation... 

```suggestion
    private async Task UseWhiteboardContextToStoreInformation... ()
```

Reply:
Changed to `UseWhiteboardForShortTermMemory`, thanks.